wp-admin/js/widgets/media-widgets.js

```diff
@@ -414,11 +399,15 @@ wp.mediaWidgets = ( function( $ ) {
 		/**
 		 * Whether a media item is selected.
 		 *
-		 * @return {boolean} Whether selected.
+		 * @return {boolean} Whether selected and no error.
 		 */
 		isSelected: function isSelected() {
 			var control = this;
 
+			if ( control.model.get( 'error' ) ) {
```

Comment:
I thought I had written some qunit tests around `isSelected` but they must have gotten lost in the shuffle on that PR.  This method, and the changes here, could use some test coverage.  I'd be happy to add that in here on this branch if you'd like.